specifyweb/workbench/upload/tests/testuploading.py

```diff
@@ -328,11 +328,13 @@ def test_attachmentimageattribute(self) -> None:
         plan = UploadTable(
             name='Attachment',
             wbcols={'guid': parse_column_options('guid')},
+            overrideScope=None,
```

Comment:
Not very familiar with Python, but is there support for default values so that you don't have to add this for all the tests?

Reply:
Fixed in 48d6367